manager/main.go

```diff
@@ -186,3 +197,24 @@ func initVaultConnection() (*api.Client, error) {
 	}
 	return vaultClient, nil
 }
+
+// This method decides based on the environment variables that are set which
+// cluster manager instance should be initiated.
+func NewClusterManager(mgr manager.Manager) multicluster.ClusterManager {
+	setupLog := ctrl.Log.WithName("setup")
+	if user, razeeLocal := os.LookupEnv("RAZEE_USER"); razeeLocal {
+		razeeURL := strings.TrimSpace(os.Getenv("RAZEE_URL"))
+		password := strings.TrimSpace(os.Getenv("RAZEE_PASSWORD"))
+		orgID := strings.TrimSpace(os.Getenv("RAZEE_ORG_ID"))
+
+		setupLog.Info("Using razee at " + razeeURL + " orgId " + orgID)
+		return razee.NewRazeeManager(strings.TrimSpace(razeeURL), strings.TrimSpace(user), password, orgID)
+	} else if apiKey, satConf := os.LookupEnv("IAM_API_KEY"); satConf {
```

Comment:
Should rename to something more concrete (IAM is not a unique indicator, it's used everywhere).